src/sentry/static/sentry/app/views/projects/projects.jsx

```diff
@@ -54,19 +96,27 @@ class Projects extends React.Component {
             onSavedSearchCreate={this.onSavedSearchCreate}
             searchPlaceholder={t('Search for projects')}
             groupOptions={groupOptions}
-            grouping={this.props.groupBy}
+            grouping={groupBy}
             onGroup={this.onGroup}
             onSearch={this.onSearch}
             orgId={this.props.organization.id}
-            query={this.props.query}
+            query={query}
           />
           <ListView
             orgId={this.props.organization.id}
             columns={this.getHeaders()}
-            dataById={this.props.projectSearchEntry.byIds}
+            dataById={byIds}
             canSelect={false}
-            visibleIds={Object.keys(this.props.projectSearchEntry.byIds)}
+            visibleIds={Object.keys(byIds)}
```

Comment:
There would be an issue with this later on if we stop clearing the `byIds` dictionary on every fetch, e.g. if we want to keep the prev. and next page in memory. I would just use `this.props.projectSearchEntry.visibleIds` and set that in the reducer, so the UI doesn't need to worry about it.

But I don't think it needs to be fixed now.